sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -22211,7 +22239,7 @@ namespace Azure.Storage.Blobs.Models
     /// <summary>
     /// the quick query body
     /// </summary>
-    public partial class QueryRequest
+    internal partial class QueryRequest
```

Comment:
The types we're hiding here - they didn't arrive to master yet right?

Reply:
Nope, they are not present in master.